lighthouse-extension/app/src/popup.js

```diff
@@ -121,6 +121,7 @@ document.addEventListener('DOMContentLoaded', _ => {
   /**
    * Generates a document fragment containing a list of checkboxes and labels
    * for the aggregation categories.
+   * @param {!Object<null>} list
```

Comment:
@brendankenny is `Object<null>` a thing? I only found https://github.com/google/closure-compiler/wiki/Annotating-JavaScript-for-the-Closure-Compiler#type-application